google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -164,28 +163,19 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           "App Engine facet not found in '" + artifact.getName() + "' artifact");
     }
 
-    CloudSdkService cloudSdkService = CloudSdkService.getInstance();
```

Comment:
I looked at the common lib code and it will check for null and still throw an AppEngineException (or a subclass of it).